CMakeLists.txt

```diff
@@ -663,6 +691,30 @@ enable_cppcheck(
         unreadVariable:*src/composable_kernel/host/*/*
         unreadVariable:*src/composable_kernel/external/*/*
         unmatchedSuppression
+        #TODO Code Quality WORKAROUND ROCm 5.3 && Ubuntu 22.04 && C++17 && cppcheck 2.9 update
```

Comment:
You should just use inline suppressions for these except the `ctu*` errors.

Reply:
@pfultz2  yes I fully agree with you. This PR does not actually "cause" any regression so I plan to create post-merge tasks on these warnings, and distribute the work among more team members to eliminate the suppressions.